tests/test_suites/ShortSystemTests/test_unlink_trash_config_hot_reload.sh

```diff
@@ -0,0 +1,124 @@
+timeout_set 4 minutes
+
+CHUNKSERVERS=4 \
+	MOUNT_EXTRA_CONFIG="sfscachemode=NEVER" \
+	MASTER_EXTRA_CONFIG="CHUNKS_LOOP_MIN_TIME = 1`
+			`|CHUNKS_LOOP_MAX_CPU = 90`
+			`|OPERATIONS_DELAY_INIT = 0" \
+	CHUNKSERVER_EXTRA_CONFIG="MAGIC_DEBUG_LOG = ${TEMP_DIR}/log`
```

Comment:
```suggestion
	CHUNKSERVER_EXTRA_CONFIG="MAGIC_DEBUG_LOG = ${TEMP_DIR}/log|LOG_FLUSH_ON=DEBUG`
```

Consider adding `DEBUG` log level to get the desired level of output.

Reply:
thanks